ingress/controllers/nginx/nginx/template.go

```diff
@@ -101,3 +103,56 @@ func toCamelCase(src string) string {
 	}
 	return string(bytes.Join(chunks, nil))
 }
+
+func buildRateLimitZones(input interface{}) []string {
```

Comment:
same comment as I made in the other pr about comments with examples of the output where possible